attractions/button/button.svelte

```diff
@@ -43,8 +44,7 @@
 {#if href}
   <a
     {href}
```

Comment:
Shouldn't the `href` be removed if `disabled == true`?

Reply:
Well...
As it currently stands, not really. The docs state that `disabled` has no effect on link buttons. We could alter that behaviour if we wanted. As of now, we simply pass through the `disabled` prop to allow the user to style disabled links if they so desire

In other words, it's a major gray area, there's a lot of ways to go about it, we need to settle on one

P.S. Removing the `href` is a bad idea because it removes the default link styling.

Reply:
I saw that the docs represent the current state of the component, but I wasn't sure if that was actually the goal or just an unintended "limitation". I believe the prop should still apply and disable the navigation. This is typically done by just removing the `href` attribute. An alternative can be to set it to `javascript:void(0)`, so it's still there but does nothing